docs_user/modules/proc_migrating-mon-from-controller-nodes-verification.adoc

```diff
@@ -9,10 +9,19 @@ After you finish migrating your Ceph Monitor daemons to the target nodes, verify
 * Verify that the {CephCluster} cluster is healthy:
```

Comment:
```suggestion
. Verify that the {CephCluster} cluster is healthy:
```
Now that we have more than 1 step, I'm changing the steps to a numbered list.

Reply:
Ack, makes sense! (Fixed)